go.mod

```diff
@@ -11,14 +11,14 @@ require (
 	github.com/pion/randutil v0.1.0
 	github.com/pion/rtcp v1.2.14
 	github.com/pion/rtp v1.8.7
-	github.com/pion/sctp v1.8.19
+	github.com/pion/sctp v1.8.22
 	github.com/pion/sdp/v3 v3.0.9
 	github.com/pion/srtp/v2 v2.0.20
 	github.com/pion/stun v0.6.1
 	github.com/pion/transport/v2 v2.2.10
 	github.com/sclevine/agouti v3.0.0+incompatible
 	github.com/stretchr/testify v1.9.0
-	golang.org/x/net v0.22.0
+	golang.org/x/net v0.27.0
```

Comment:
this required a `go mod tidy -go=1.16 && go mod tidy -go=1.17` which is nonsense to me